project/plugins.sbt

```diff
@@ -1,12 +1,7 @@
-resolvers += Resolver.url(
-  "HMRC Sbt Plugin Releases",
-  url("https://dl.bintray.com/hmrc/sbt-plugin-releases")
-)(Resolver.ivyStylePatterns)
-resolvers += "HMRC Releases" at "https://dl.bintray.com/hmrc/releases"
-resolvers += "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases/"
+resolvers += MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefacts.tax.service.gov.uk/maven2")
+resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.12.0")
-addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.2.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.0.0")
 addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.1.0")
 addSbtPlugin("uk.gov.hmrc" % "sbt-artifactory" % "1.12.0")
```

Comment:
I believe with `sbt-auto-build - 3.0.0`, you can remove  the `sbt-artifactory` plug -in... see https://confluence.tools.tax.service.gov.uk/display/TEC/2021/05/05/New+version+of+sbt-auto-build

Reply:
Thanks for pointing it our.
I just pushed and amended commit